### DISCUSSION TITLE
links behind toasts are not clickable

### DISCUSSION THREAD

**Discussion by Qteam-tech** [twbs/bootstrap#35072] [Q&A] [answered]
Hi everyone,
I used a toast in my website and figure out that the links come behind the toast not working until you scroll the page and come out the position that you set the toast will appear. 
for example if you set to to <div class="position-fixed bottom-0 end-0 p-3" style="z-index: 11">  that mean right down side of screen, the links and every thing behind that position will not clickable.
I hope you understand what I mean.
thanks so much for your great work

**Comment by ghost** [comment]
As far as I can see, z-index css element is the problem or solution. I can't be sure unless I see myself.

**Comment by Qteam-tech** [comment]
Thanks for your reply.
Actually, I used z-index element in my code but still there is a problem.
this is my code: 
div class="position-fixed bottom-0 start-0 p-3" style="z-index: 100"

**Comment by Qteam-tech** [comment, marked as answer]
I figure out where is the problem or solution. 
I found in bootstrap css code 
`.toast:not(.showing):not(.show) {
  opacity: 0;
}`
So changed it with 
`.toast:not(.showing):not(.show) {
  display: none;
}`
and work fine. 
thanks all
